templates/zerver/emails/password_reset.txt

```diff
@@ -1,9 +1,12 @@
-{% if attempted_realm %}
+{% if no_account_in_realm %}
 Someone (possibly you) requested a password reset email for {{ email }} on
-{{ attempted_realm.uri }}, but {{ email }} does not
-have an active account in {{ attempted_realm.uri }}.  However, {{ email }} does
+{{ realm_uri }}, but {{ email }} does not
+have an active account in {{ realm_uri }}.
+{% if account_exists_another_realm %}
+However, {{ email }} does
```

Comment:
This line is too small compared with line no:2 and next line.

Reply:
Want to push a branch that revised this?  Better to just debug it interactively.